src/main/java/com/synopsys/integration/blackduck/rest/ApiTokenBlackDuckHttpClient.java

```diff
@@ -76,7 +76,7 @@ public final Response attemptAuthentication() throws IntegrationException {
         Map<String, String> headers = new HashMap<>();
         headers.put(AuthenticationSupport.AUTHORIZATION_HEADER, "token " + apiToken);
         // https://github.com/blackducksoftware/blackduck-common/issues/268
-        headers.put("Content-Length", "0");
+//        headers.put("Content-Length", "0");
```

Comment:
Is this no longer needed?

Reply:
It was causing errors with http client 5. We can discuss what to do.